Makefile

```diff
@@ -14,8 +14,10 @@ tmpdir = tmp
 debugdir = tmp/debug
 bindir = bin
 
-CXXFLAGS = -std=c++17 -I$(srcdir) -I$(resdir) $(shell fltk-config --use-images --cxxflags)
-LDFLAGS = $(shell fltk-config --use-images --ldflags) $(shell pkg-config --libs libpng xpm)
+fltk-config = $(bindir)/fltk-config
+
+CXXFLAGS = -std=c++17 -I$(srcdir) -I$(resdir) $(shell $(fltk-config) --use-images --cxxflags)
+LDFLAGS = $(shell $(fltk-config) --use-images --ldflags) $(shell pkg-config --libs libpng xpm)
```

Comment:
This change to the Makefile avoids the need for `export PATH="$PWD/bin:$PATH"` in INSTALL.md and .github/workflows/main.yml